bin/obj2gltf.js

```diff
@@ -1,53 +1,136 @@
-    console.log('  -s --separate           Writes out separate geometry/animation data files, shader files, and textures instead of embedding them in the glTF file.');
-    console.log('  -t --separateImage      Write out separate textures only.');
-    console.log('  -c --compress           Quantize positions, compress texture coordinates, and oct-encode normals.');
-    console.log('  -h, --help              Display this help');
-    console.log('      --ao                Apply ambient occlusion to the converted model');
-    console.log('      --cesium            Optimize the glTF for Cesium by using the sun as a default light source.');
-    process.exit(0);
-}
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+var args = process.argv;
+args = args.slice(2, args.length);
 
-var objFile = defaultValue(argv._[0], defaultValue(argv.i, argv.input));
-var outputPath = defaultValue(argv._[1], defaultValue(argv.o, argv.output));
-var binary = defaultValue(defaultValue(argv.b, argv.binary), false);
-var separate = defaultValue(defaultValue(argv.s, argv.separate), false);
-var separateImage = defaultValue(defaultValue(argv.t, argv.separateImage), false);
-var compress = defaultValue(defaultValue(argv.c, argv.compress), false);
-var ao = defaultValue(argv.ao, false);
-var optimizeForCesium = defaultValue(argv.cesium, false);
-
-if (!defined(objFile)) {
-    throw new Error('-i or --input argument is required. See --help for details.');
+var argv = yargs
+    .usage('Usage: node $0 -i inputPath -o outputPath')
+    .example('node $0 -i ./specs/data/box/box.obj -o box.gltf')
+    .help('h')
+    .alias('h', 'help')
+    .options({
+        input : {
+            alias: 'i',
+            describe: 'Path to the obj file.',
+            type: 'string',
+            normalize: true
+        },
+        output : {
+            alias: 'o',
+            describe: 'Path of the converted glTF file.',
+            type: 'string',
+            normalize: true
+        },
+        binary : {
+            alias: 'b',
+            describe: 'Save as binary glTF.',
+            type: 'boolean',
+            default: false
+        },
+        separate : {
+            alias: 's',
+            describe: 'Write separate geometry data files, shader files, and textures instead of embedding them in the glTF.',
+            type: 'boolean',
+            default: false
+        },
+        separateTextures : {
+            alias: 't',
+            describe: 'Write out separate textures only.',
+            type: 'boolean',
+            default: false
+        },
+        compress : {
+            alias: 'c',
+            describe: 'Quantize positions, compress texture coordinates, and oct-encode normals.',
+            type: 'boolean',
+            default: false
+        },
+        optimize : {
+            alias: 'z',
+            describe: 'Use the optimization stages in the glTF pipeline.',
```

Comment:
From an "end user reading documentation" standpoint, this sentence means nothing.  No need to mention the glTF pipeline, either state exactly what optimizations are performed OR simply state "optimized the model for size and speed" or something similar.